cmds/tui.c

```diff
@@ -2463,6 +2469,18 @@ static void tui_main_loop(struct opts *opts, struct uftrace_data *handle)
 				full_redraw = true;
 			}
 			break;
+		case 's':
+			if (!tui_window_change(win, &report->win)) {
+				curr_sort_key = (curr_sort_key + 1) %
+					ARRAY_SIZE(report_sort_key);
+				report_setup_sort(report_sort_key[curr_sort_key]);
+				report_sort_nodes(&tui_report.name_tree,
+						&tui_report.sort_tree);
```

Comment:
Please give two more whitespaces to align with the previous line.
```suggestion
						  &tui_report.sort_tree);
```